server/@types/express/index.d.ts

```diff
@@ -46,6 +46,7 @@ declare module 'express-session' {
     previousWorkExperienceTypesForm: PreviousWorkExperienceTypesForm
     previousWorkExperienceDetailForm: PreviousWorkExperienceDetailForm
     affectAbilityToWorkForm: AffectAbilityToWorkForm
+    workInterestRolesForm: WorkInterestRolesForm
```

Comment:
We need to add the `WorkInterestRolesForm` to the session - it was the only form that was not on the session previously because up until now it didnt have any validation at all 😬 so there was no need to hold it on the session.

This PR introduces validation for this form (below), hence we now need to be able to hold it on the session